src/contexts/CubeContext.js

```diff
@@ -54,6 +54,18 @@ const getDetails = async (cardId) => {
   return json.card;
 };
 
+const canUserEdit = (cube, user) => {
```

Comment:
There are a lot of editable things in the cube, and if we're going to restrict this to only editing the list, I think it should be reflected in the name to prevent confusion.

Reply:
Yeah, that makes sense. I'd noticed that and was thinking I should add more nuances here. Perhaps I should add a `userCubeRole` field to the context that's one of `"owner"` `"collaborator"` or `"viewer"` and then update places that use canEdit to key off of that instead. Some would require "owner", others "owner" or "collaborator".

Reply:
Also sorry I missed the bit in CONTRIBUTING.md about the `[WIP]` prefix on the PR. I've added that now

Reply:
That sounds like a solid approach if you want to spend the time on it. Otherwise, just using two booleans instead of one should also work fine.